aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -0,0 +1,172 @@
+
+const columns = [
+  {
+    property: 'name',
+    header: 'Name',
+    size: 'small',
+    primary: true,
+  },
+  {
+    property: 'rocket',
+    header: 'Rocket',
+    size: 'small',
+    render: datum => <Text key={datum.rocket.name}>{datum.rocket.name}</Text>,
+    align: 'end',
+  },
+  {
+    property: 'success',
+    header: 'Success',
+    size: 'xsmall',
+    align: 'end',
+    sortable: false,
+    render: datum => {
+      if (datum.success === false) {
+        const content = (
+          <Box width={{ max: 'medium' }}>
+            {datum.failures?.map(({ reason }) => (
+              <Text key={reason}>{reason}</Text>
+            ))}
+          </Box>
+        );
+        return (
+          <Tip
+            plain
+            content={content}
+            dropProps={{
+              round: 'medium',
+              pad: 'small',
+              background: 'background-back',
+            }}
+          >
+            <Box>
+              <StatusCritical color="red" />
+            </Box>
+          </Tip>
+        );
+      }
+      return undefined;
+    },
+  },
+];
+
+export const PaginationServerTableExample = () => {
+  const [sort, setSort] = useState({ property: 'name', direction: 'asc' });
+  const [data, setData] = useState([]);
+  const [numberItems, setNumberItems] = useState(0);
+  const [page, setPage] = useState(1);
+  const [limit, setLimit] = useState(10);
+  const pageResultStart = (page - 1) * limit + 1;
+  const pageResultEnd = Math.min(page * limit, numberItems);
+  const options = [5, 10, 25, 50, 100];
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const query = {
+        options: {
+          populate: [
+            {
+              path: 'rocket',
+              select: { name: 1 },
+            },
+          ],
+          sort: {
+            [sort.property || 'name']: sort.direction || 'asc',
+          },
+          select: ['name', 'success', 'failures'],
+          limit,
+          page,
+        },
+      };
+      fetch('https://api.spacexdata.com/v4/launches/query', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(query),
+      })
+        .then(response => response.json())
+        .then(d => {
+          setPage(d.page);
+          setNumberItems(d.totalDocs);
+          setData(d.docs || []);
+        })
+        .catch(error => console.error('Unable to get data:', error));
+    };
+    fetchData();
+  }, [sort, page, limit]);
+
+  return (
+    <Box pad="small">
+      <Heading id="server-side-pagination-heading" level={3}>
+        Launches
+      </Heading>
+
+      {numberItems > limit && (
+        <Box
+          pad={{ vetical: 'small' }}
+          align="center"
+          direction="row-responsive"
+          gap="small"
+        >
+          <Text>Select</Text>
+          <Select
+            id="select"
+            name="select"
+            placeholder="10"
```

Comment:
These aren't needed. You're not in a form and the `value` is always set so you'll never see the placeholder

Reply:
Should I put the Select component in a Form/FormField component? I'm getting a WAVE check "missing form label" error, and I've noticed that the Select components on the guidance page [here](https://design-system.hpe.design/components/select) are wrapped in a Form component.

Reply:
No reason to have it within a Form let alone a FormField. FormField provides label and borders plus some validation and such when used in a Form, none of which is needed here. And you never need to do a form submit or any of the other logic related to a form so I think WAVE check is overly opinionated.